README.md

```diff
@@ -1,3 +1,5 @@
+[![codecov](https://codecov.io/gh/osrf/romi-dashboard/branch/wip/graph/badge.svg)](https://codecov.io/gh/osrf/romi-dashboard)
```

Comment:
is this right? shouldn't it be master branch?

Reply:
Updated.